.vscode/setting.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.formatOnType": true,
```

Comment:
format on type sometimes messes things up. I'd prefer to turn it off by default.